packages/viteval/package.json

```diff
@@ -11,15 +11,15 @@
     "@viteval/cli": "workspace:*",
     "@viteval/core": "workspace:*",
     "@viteval/internal": "workspace:*",
-    "@viteval/ui": "workspace:*"
+    "@viteval/ui": "workspace:*",
+    "vitest": "^3.2.4"
```

Comment:
we should not compile this down but instead rely, still need to assess if should be peer dep